### PR TITLE
IssueFour, Fetching and Mapping

### PR DESCRIPTION
**Pull request by hamedbahrami479**
Would be great, if someone could have a look. I am afraid there is still something weird.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
-      <Header />
-      <Card />
+      <Title />
+      {characters.map((character) => (
+        <Card
+          key={character.id}
+          image={character.image}
+          name={character.name}
+        />
+      ))}
```
Comment (mbosselmann) [inline src/App.js:30]: You need to add the `ul` element in your App and not in your `Card`.

```
<ul role="list">
      {characters.map((character) => (
        <Card
          key={character.id}
          image={character.image}
          name={character.name}
        />
      ))}
</ul>
```
If you add the `ul` element in your `Card` component then every card has an own `ul` but we want to have all cards in one `ul`. I hope this makes it clear now, as I think my explanation before was misunderstanding. 🙏

File: src/components/Card.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
 const Card = () => {
   return (
-    <div>
-    <CardItem>
-      <img src="https://rickandmortyapi.com/api/character/avatar/2.jpeg" alt="Hier muss ein Bild sein" />
-      <h2> Morty Smith </h2>
-    </CardItem>
-    </div>
+    <ul role="list">
```
Comment (mbosselmann) [inline src/components/Card.js:6]: And here you should remove the `ul` element. :)

**Review by mbosselmann — COMMENTED** [review]
Looks good! 👍 I have only one remark about the `ul` element that you should change. I hope my explanation is better to understand now. If not we can also discuss this.

File: src/App.js (hunk 1 above)
Comment (hamedbahrami479) [reply src/App.js:30]: thank u so much. I would say the confusion was not made by ur explanation rather than by my leck of experienced. :)

**Pull request merged**